src/GitUtilities.js

```diff
@@ -40,7 +40,7 @@ export default class GitUtilities {
 
   @logger.logifySync
   static pushWithTags(tags) {
-    ChildProcessUtilities.execSync("git push");
+    ChildProcessUtilities.execSync("git push origin " + GitUtilities.getCurrentBranch());
```

Comment:
Not a bad assumption, but we assume the branch is always `origin` in these 2 lines - I guess that could go in config (https://github.com/kittens/lerna/issues/42)?